api/grpc/server.go

```diff
@@ -50,17 +51,27 @@ type PayChServer struct {
 	chUpdatesNotif   map[string]map[string]chan bool
 }
 
-// NewPayChServer returns a new grpc server that can server the payment channel API.
-func NewPayChServer(n perun.NodeAPI) *PayChServer {
-	return &PayChServer{
+// ListenAndServePayChAPI starts a payment channel API server that listens for incoming grpc
+// requests at the specified address and serves those requests using the node API instance.
+func ListenAndServePayChAPI(n perun.NodeAPI, grpcPort string) error {
```

Comment:
Function name looks odd, since it is having 2 different functions in one name, either we can keep separate functions or change the function name.

Reply:
`ListenAndServeXYZ` is a common function in server packages, you can find this name in even in the http package in standard library, [here](https://golang.org/pkg/net/http/?m=all#ListenAndServe).

Reply:
Ok.